dnx/src/Microsoft.DotNet.xunit.performance.runner.dnx/project.json

```diff
@@ -12,7 +12,7 @@
   "requireLicenseAcceptance": false,
 
   "dependencies": {
-    "Microsoft.DotNet.xunit.performance.run.core": "99.99.99-dev",
+    "Microsoft.DotNet.xunit.performance.run.core": "1.0.0-alpha-build0018",
```

Comment:
If you go back through the history of xunit.performance.msbuild, you'll see that it used to include this snippet:
```xml
    <RegexReplace
        Pattern='99\.99\.99-dev'
        Replacement='$(BuildSemanticVersion)'
        Files='@(NuspecFiles);@(ProjectJsonFiles)'
        Condition=" '$(BuildSemanticVersion)' != '' "/>
```

I think the "ProjectJsonFiles" part needs to be put back (along with the definition of ProjectJsonFiles). I don't think we need the NuspecFiles part since the version number is passed directly on the "nuget pack" command line.

I admit this is tricky.